templates/utils.html

```diff
@@ -94,7 +94,13 @@
 	</p>
 	<!-- POST MEDIA/THUMBNAIL -->
 	{% if (prefs.layout.is_empty() || prefs.layout == "card") && post.post_type == "image" %}
-	<a href="{{ post.media.url }}" class="post_media_image {% if post.media.height / post.media.width < 2 %}short{% endif %}" >
+
+	<!-- reddit-hosted images do not provide us with width/height information: the values will be 0. -->
+	<!-- To work around "content reflow" problems, use svg tags to keep the image in place. -->
+	<!-- This causes two problems: right-clicking images will have a very degraded menu: no "copy image (link)", and bad display of reddit-hosted previews. -->
+	<a href="{{ post.media.url }}" class="post_media_image {% if post.media.height < post.media.width*2 %}short{% endif %}" >
+		{% if post.media.height == 0 || post.media.width == 0 %}
+		<!-- debug: reddit-hosted: svg workaround image thumbnail -->
 		<svg
 			width="{{ post.media.width }}px" 
```

Comment:
Need to play around with what we can put here because we do not have width/height (they are 0 in this path)